app/views/movies/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= @movie.title %>
+<%= @movie.creation_year %>
+<%= @movie.genre %><br>
+<% @movie.actors.order_by_age.each do |actor| %>
+<%= actor.name %>
+<%= actor.age %><br>
+<%end%>
+<div id="average age of actors">
+  <h2><%= @movie.actors.average_age %></h2>
+</div>
+
+<%= form_with url: "/movies/#{@movie.id}", method: :patch, data: {turbo: false} do |f| %>
```

Comment:
This page and the form looks fine, again I recommend swapping out your method for `:post` isntead, since we're creating a relationship.